api/v1alpha1/dspipeline_types.go

```diff
@@ -330,7 +330,6 @@ type ExternalStorage struct {
 
 type S3CredentialSecret struct {
 	// +kubebuilder:validation:Required
-	// Note: In V2 this value needs to be mlpipeline-minio-artifact
```

Comment:
Probably should update this to a description of what SecretName should be (so the CRD has a useful description field) rather than totally remove it

Reply:
Done.